### ISSUE TITLE
Cannot Export Context

### ISSUE THREAD

**Issue by epotter2297** [ecyrbe/zodios-express#41]
```ts
import { zodiosContext } from '@zodios/express';
import z from 'zod';

const user = z.object({
  user: z
    .object({
      id: z.number(),
      name: z.string(),
      isAdmin: z.boolean(),
    })
    .optional(),
});

export const ctx = zodiosContext(user);
```

I'm trying to export `ctx` in a similar way to how it's done in the next.js example, but I'm getting an error that says `Property 'context' of exported class expression may not be private or protected. ts(4094)`.

**Comment by ecyrbe** [comment]
You need to not emit `.d.ts` files (if you are not a library aithor, no reason to emit them) so set  "declaration": false` in your tsconfig, it's an issue with typescript d.ts hability to represent private fields in classes when exporting d.ts files.

Since this behaviour is bad, if you are a library author, you can also also wait for version 9 of `@zodios/express`

**Comment by ecyrbe** [comment]
fixed in version v0.3.4
